go.mod

```
@@ -306,3 +306,7 @@ require (
 
 // See https://github.com/moby/moby/issues/42939#issuecomment-1114255529
 replace github.com/docker/docker => github.com/docker/docker v20.10.3-0.20220224222438-c78f6963a1c0+incompatible
+
+// v1.2.0 is taken from github.com/open-policy-agent/opa v0.42.0
+// v1.2.0 incompatible with github.com/docker/docker v20.10.3-0.20220224222438-c78f6963a1c0+incompatible
+replace oras.land/oras-go v1.2.0 => oras.land/oras-go v1.1.1
```

We should remove the version so that the newer version will also be replaced.

```suggestion
replace oras.land/oras-go => oras.land/oras-go v1.1.1
```